web/src/toga_web/widgets/scrollcontainer.py

```diff
@@ -0,0 +1,78 @@
+from .base import Widget
+
+
+class ScrollContainer(Widget):
+    def __init__(self, interface):
+        super().__init__(interface)
+        self.native.addEventListener("scroll", self._on_native_scroll)
+
+    def create(self):
+        self._horizontal_enabled = True
+        self._vertical_enabled = True
+
+        self.native = self._create_native_widget(
+            "div",
+            classes=["toga-scroll-container"],
```

Comment:
Why is this extra class required? The widget will automatically have the classes `toga` and `scrollcontainer` added. For consistency with `Box`, it might be worth adding `container`, but `toga-scroll-container` seems redundant.